evap/static/scss/_adjustments.scss

```diff
@@ -140,6 +140,18 @@ $alert-colors: (
     white-space: normal;
 }
 
+.user-multi-select {
```

Comment:
Technically, this class just makes the multi select have one entry per line. We have some other user multi selects that will not use this class, so it might make sense to have the name reflecting that.